packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -453,24 +455,8 @@ export async function handleSocialLoginCallback({
   try {
     // Sign a session
     if (isCosmos && desiredChain) {
-      // Not every chain prefix will succeed, so Magic defaults to osmo... as the Cosmos prefix
       const bech32Prefix = desiredChain.bech32Prefix;
-      try {
-        magicAddress = await magic.cosmos.changeAddress(bech32Prefix);
```

Comment:
Just going to use `cosmos` addresses instead of individual chains because this is verifiable from magic.